l10n_it_website_sale_fiscalcode/README.rst

```diff
@@ -0,0 +1,99 @@
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--italy-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_website_sale_fiscalcode
+    :alt: OCA/l10n-italy
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-italy-12-0/l10n-italy-12-0-l10n_it_website_sale_fiscalcode
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/122/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+**Italiano**
+
+Questo modulo aggiunge il codice fiscale nel form del checkout.
+
+**English**
+
+This module adds fiscal code in the checkout form.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+**Italiano**
+
+Per utilizzare questo modulo:
+
+* Andare nel negozio;
+* Comprare qualcosa;
+* Aggiungere il codice fiscale nel form del checkout.
+
+**English**
+
+To use this module, you need to:
+
+* Go to Shop;
+* Buy something;
+* Add fiscal code in the checkout form.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-italy/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/l10n-italy/issues/new?body=module:%20l10n_it_website_sale_fiscalcode%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Agile Business Group
+
+Contributors
+~~~~~~~~~~~~
+
+* Nicola Malcontenti <nicola.malcontenti@agilebg.com>
+* Alessio Gerace <alessio.gerace@agilebg.com>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/l10n-italy <https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_website_sale_fiscalcode>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```

Comment:
Update references to v14

for example: `https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_website_sale_fiscalcode`
 --> `https://github.com/OCA/l10n-italy/tree/14.0/l10n_it_website_sale_fiscalcode`

Reply:
Thank you!